tools/hive-bigquery/hive_to_bigquery/__main__.py

```diff
@@ -17,17 +17,17 @@
 
 from google.api_core import exceptions
 
-from bigquery_component import BigQueryComponent
-from bigquery_table import BigQueryTable
-import custom_exceptions
-from gcs_storage_component import GCSStorageComponent
-from hive_component import HiveComponent
-from hive_table import HiveTable
-import kms_component
-from mysql_component import MySQLComponent
-from properties_reader import PropertiesReader
-from resource_validator import ResourceValidator
-import init_script
+from .bigquery_component import BigQueryComponent

```

Comment:
I'm not familiar with leading dots in import statements like this.
could this cause relative directory problems?
Is this the same as `hive_to_bigquery.bigquery_component`?

Typically for projects with many submodules (e.g. airflow / beam) I'd see the package name spelled out explicitly.

Reply:
Certain programmers choose relative imports vs absolute imports.  Even though relative imports are acceptable, [general guidelines](https://pep8.org/#imports) is to use absolute imports unless there is a deep nested package structure.

Reply:
Updated in e831f64